src/content/changes/highlights/2022-09-15.md

```diff
@@ -0,0 +1,31 @@
+---
+date: 2022-09-15T10:00:00
+title: Highlights for the week ending September 15, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+
+
+## User interfaces
+
+In [kubectl-gs](https://github.com/giantswarm/kubectl-gs/releases/tag/v2.22.0) we have deprecated some flags in three commands, to avoid name conflicts with global flags (which are usable in all commands). You will see a warning when using the deprecated flags. This affects the following commands:
+
+- `kubectl gs template app`: flag `--target-namespace` replaces `--namespace`, `--cluster-name` replaces `--cluster`.
+- `template catalog`: flag `--target-namespace` replaces `--namespace`.
+- `gitops add app`: flag `--target-namespace` replaces `--namespace`.
+
+Please move any scripts to the new flags, as the old flags will change their meaning with the next major release of kubectl-gs.
+
+kubectl-gs now also provides a command `get organizations`, which is a way for all management API users to list the organizations they have access to. `kubectl get organizations` in contrast requires admin permissions.
+
+## Apps
+
+- [trivy-app](https://github.com/giantswarm/trivy-app) version [v0.5.0](https://github.com/giantswarm/trivy-app/blob/main/CHANGELOG.md#050---2022-09-09) updates to Trivy version `0.29.2`, introducing support for Trivy modules. The `spring4shell` module is installed by default.
+
+
+## Documentation
```

Comment:
```suggestion
## Documentation
- We added the [Documentation](https://docs.giantswarm.io/ui-api/observability/grafana/custom-dashboards/) how to package custom dashboards together with apps instead of packaging them into a custom repository. 
```